Source/WebKit/Platform/IPC/StreamClientConnection.h

```diff
@@ -72,6 +72,7 @@ class StreamClientConnection final : public ThreadSafeRefCounted<StreamClientCon
     void setMaxBatchSize(unsigned);
 
     void open(Connection::Client&, SerialFunctionDispatcher& = RunLoop::current());
+    Error flushSentMessages(Timeout);
```

Comment:
Should there be a caller of this outside of just the new tests?

Reply:
Yeah, but I left it for new code as it'd be maybe easier to review the other code that way